lib/menus/darwin.json

```diff
@@ -2,7 +2,7 @@
   "menu": [{
     "label": "google-music-electron",
     "submenu": [{
-      "label": "About google-music-electron",
+      "label": "About Google Music Electron",
```

Comment:
I hadn't realized that I was so inconsistent about when I used `google-music-electron` vs "Google Music Electron". Can we hold off on these changes and take care of them in their own PR (we prob need to take an itinerary and also handle things like newsletters as well)